src/modules/reguser/RegistrationWizard.cpp

```diff
@@ -102,14 +108,13 @@ RegistrationWizard::RegistrationWizard(const QString &startMask,KviRegisteredUse
 
 	// PAGE 2
 
-
 	m_pPage2 = new QWidget(this);
-	m_pPage2Layout = new QGridLayout( m_pPage2 );
-	m_pPage2Layout->setSpacing( 4 );
-	m_pPage2Layout->setMargin( 8 );
+	m_pPage2Layout = new QGridLayout(m_pPage2);
+	m_pPage2Layout->setSpacing(4);
+	m_pPage2Layout->setMargin(8);
 
 	m_pLabel2 = new QLabel( m_pPage2);
-	m_pLabel2->setText(__tr2qs_ctx("<p>A registered user is identified by one or more <b>IRC masks</b>.<br>A mask must be in the following form:<br><b>nickname!username@hostname</b><br>and can contain wildcard characters '*' and '?'. Be careful in choosing the masks, as they are the only way to verify the identity of a registered user.<br><br>You can enter at most two masks here, if you wish to add more masks, use the \"<b>Edit</b>\" button in the Registered Users dialog. You must enter at least one mask.</p>","register"));
+	m_pLabel2->setText(__tr2qs_ctx("<p>A registered user is identified by one or more <b>IRC masks</b>.<br>A mask must be in the following form:<br><b>nickname!username@hostname</b><br>and can contain wildcard characters '*' and '?'. Be careful in choosing the masks, as they are the only way to verify the identity of a registered user.<br><br>You can enter at most two masks here, if you wish to add more masks, use the \"<b>Edit</b>\" button in the Registered Users dialog. You must enter at least one mask.<br><br><br><br><br><br></p>","register"));
```

Comment:
I managed to replace the ```<br>```'s on page 1 and 2 but page 4 is giving quite a few problems.

Some help/tips would be nice.